### PR TITLE
[libalkimia] new port

### PR DESCRIPTION
**Pull request by wrobelda**
**Describe the pull request**
  Adds new LibAlkimia port. Note that it currently uses the HEAD snapshot, since it has many fixes currently unreleased. 
  Depends on:
  - [x] #20205 
  - [x] #20198 
  - [x] #19756 
  - [x] #20210 
 
- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All official, plus `arm64-osx`

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

**Comment by BillyONeal** [comment]
There's a problem with CI caused by tensorflow escaping our job but there are legitimate regressions here.

**Comment by BillyONeal** [comment]
Please excuse the deprecated functions notification, We're trying to root out a bug in https://github.com/microsoft/vcpkg/pull/20142

**Review by PhoebeHui — COMMENTED** [review]
Thanks for your contribution!

File: ports/libalkimia/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  ],
+  "features": {
+    "tools": {
+      "description": "Enables tools, including Online Quotes Editor"
+    },
+    "webengine": {
+      "description": "Enables JavaScript support for parsing feature-rich websites",
+      "dependencies": [
+        "qt5-webengine"
+      ]
```
[the remark this answers is not in the document]
Comment (PhoebeHui) [reply ports/libalkimia/vcpkg.json:44]: @wrobelda, I'm sorry for late response!

I have tested the all features with x64-windows, it has problems in performing post-build validation, it looks lacks of dependency for ‘tools’ features?

```
-- Performing post-build validation
The following EXEs were found in /bin or /debug/bin. EXEs are not valid distribution targets.

    F:/vcpkg/20207/vcpkg/packages/libalkimia_x64-windows/bin/onlinequoteseditor5.exe

The following EXEs were found in /bin or /debug/bin. EXEs are not valid distribution targets.

    F:/vcpkg/20207/vcpkg/packages/libalkimia_x64-windows/debug/bin/onlinequoteseditor5.exe

Running command:
   "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools/MSVC\14.29.30133\bin/HostX86/x86\dumpbin.exe" /exports "F:\vcpkg\20207\vcpkg\packages\libalkimia_x64-windows\debug/bin\alkimia5-8.dll"
 failed
```

[the remark this answers is not in the document]
Comment (wrobelda) [reply ports/libalkimia/vcpkg.json:44]: Thanks. `AUTO_CLEAN` was missing in `vcpkg_copy_tools()`. Fixed.

**Review by PhoebeHui — APPROVED** [review]
All features test passed with x64-windows.

File: ports/libalkimia/vcpkg.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (PhoebeHui) [reply ports/libalkimia/vcpkg.json:44]: Great!

**Pull request merged**